experimental/Schemes/Resolution_structure.jl

```diff
@@ -0,0 +1,754 @@
+  all_entries = elem_type(OO(W))[trans_mat[i, j] for (i, j) in all_entries_ind]
+  entry_id = ideal(OO(W), all_entries)
+  lambda = coordinates(one(OO(W)), entry_id)
+
+  non_zero_entries = [k for k in 1:length(lambda) if !is_zero(lambda[k])]
+
+  loc_results = Tuple{<:AbsAffineScheme, Vector{Int}, Vector{Int}, <:RingElem}[]
+  for k in non_zero_entries
+    i, j = all_entries_ind[k]
+    h_ij = trans_mat[i, j]
+    U_ij = PrincipalOpenSubset(W, OO(W)(h_ij))
+    res_mat = change_base_ring(OO(U_ij), trans_mat) # TODO: Avoid checks here
+    new_row_ind = vcat(row_ind, [i])
+    new_col_ind = vcat(col_ind, [j])
+    
+    # Do Gaussian elimination on the matrix to kill off the other entries in this row
+    u = res_mat[i, j]
+    inv_u = inv(u)
+    for l in 1:r
+      l in new_col_ind && continue
+      res_mat = add_column!(res_mat, -inv_u * res_mat[i, l], j, l)
+    end
+    loc_results = vcat(loc_results, 
+          find_refinement_with_local_system_of_params_rec(
+              U_ij, mod_gens, new_row_ind, new_col_ind, res_mat; check=check
+             )
+         )
+  end
+  return loc_results
+end
+
+##################################################################################################
+#  locus of order at least b and of maximal order
+##################################################################################################
+
+function max_order_locus(I::AbsIdealSheaf)
+  return _delta_list(I)[end]
+end
+
+function locus_of_order_geq_b(I::AbsIdealSheaf, b::Int)
+  return _delta_list(I,b)[end]
+end
+
+function _delta_ideal_for_order(inc::CoveredClosedEmbedding)
+
+  W = codomain(inc)
+  I = image_ideal(inc)
+
+  Delta_dict = IdDict{AbsAffineScheme,Ideal}()
+
+  for U in affine_charts(W)
+    XU, inc_U = sub(U,I(U))
+    Cov,Chart_dict = Oscar.find_refinement_with_local_system_of_params(U)
+    Delta_dict[U] = (Oscar._delta_ideal_for_order(Oscar.CoveredClosedEmbedding(inc_U), Cov, Chart_dict))(U)
+  end
+
+  return IdealSheaf(W,Delta_dict;check=false)
+end
+
+function _delta_list(inc::CoveredClosedEmbedding)
+  I = image_ideal(inc)
+  return _delta_list(I)
+end
+
+function _delta_list(I::AbsIdealSheaf, b::Int=0)
+  W = scheme(I)
+  is_smooth(W) || error("ambient scheme needs to be smooth")
+  Delta_list = AbsIdealSheaf[]
+  j = 0
+  while (( !is_one(I) && b == 0 ) || ( j < b ))
+    push!(Delta_list, I)
+    inc = Oscar.CoveredClosedEmbedding(scheme(I),I)
+    I = Oscar._delta_ideal_for_order(inc)
+    j = j + 1
+  end
+
+  return Delta_list
+end
+
+function _delta_ideal_for_order(inc::CoveredClosedEmbedding, Cov::Covering, 
+       ambient_param_data::IdDict{<:AbsAffineScheme,
+                                 <: Tuple{Vector{Int64},Vector{Int64},<:RingElem}};
+       check::Bool=true)
+
+  W = codomain(inc)                                
+  @check is_smooth(W) "codomain of embedding needs to be smooth"
+#  @check is_equidimensional(W) "codomain of embedding needs to be equidimensional"
```

Comment:
Is this still disabled intentionally?

Reply:
Yes, because this is not systematically set right now.